### PR TITLE
Fix IDEA-244645

### PR DESCRIPTION
**Pull request by dillon-giacoppo**
https://youtrack.jetbrains.com/issue/IDEA-244645

I can't create new Java classes in 2020.2 without errors as described in ticket. Really annoying but fix seems simple enough.

### CODE REVIEW

**Review by uri-canva — APPROVED** [review]
Nice! Have you sent this fix upstream too?

**Comment by dillon-giacoppo** [comment]
Not yet, my understanding was that upstream doesn't accept fixes because its a copybara export. I can raise the issue 🤷

**Comment by uri-canva** [comment]
They accept fixes, they just merge them manually, not through github.

**Comment by mskonovalov** [comment]
yeah, send it upstream anyway

**Comment by uri-canva** [comment]
@dillon-giacoppo did you send this upstream already? Should I do it?

**Comment by dillon-giacoppo** [comment]
@uri-canva I had forgotten, raised https://github.com/bazelbuild/intellij/pull/2128 (and related issue). It will require a bit more effort to get this upstream since it requires a compatibility class. I don't have the time to do that investigation so will ice it until maintainers approve.

**Pull request merged**
